.gitmodules

```diff
@@ -14,3 +14,9 @@
 	path = extern/ParseOggVorbis
 	url = https://github.com/albertz/ParseOggVorbis.git
 	branch = master
+[submodule "extern/warprnnt/warp-transducer"]
+	path = extern/warprnnt/warp-transducer
+	url = https://github.com/HawkAaron/warp-transducer.git
```

Comment:
This is twice here now. This first submodule can be removed then.

Reply:
oops, fixed.